lib/apis/v3/felixconfig.go

```diff
@@ -46,40 +46,40 @@ type FelixConfigurationSpec struct {
+	// IptablesPostWriteCheckInterval is the period, in seconds, after Felix has done a write
 	// to the dataplane that it schedules an extra read back in order to check the write was not
 	// clobbered by another process. This should only occur if another application on the system
 	// doesn’t respect the iptables lock. [Default: 1]
-	IptablesPostWriteCheckIntervalSecs *int `json:"iptablesPostWriteCheckIntervalSecs,omitempty"`
+	IptablesPostWriteCheckInterval *metav1.Duration `json:"iptablesPostWriteCheckIntervalSecs,omitempty" configv1timescale:"seconds" confignamev1:"IptablesPostWriteCheckIntervalSecs"`
 	// IptablesLockFilePath is the location of the iptables lock file. You may need to change this
 	// if the lock file is not in its standard location (for example if you have mapped it into Felix’s
 	// container at a different path). [Default: /run/xtables.lock]
 	IptablesLockFilePath string `json:"iptablesLockFilePath,omitempty"`
-	// IptablesLockTimeoutSecs is the time, in seconds, that Felix will wait for the iptables lock,
+	// IptablesLockTimeout is the time, in seconds, that Felix will wait for the iptables lock,
 	// or 0, to disable. To use this feature, Felix must share the iptables lock file with all other
 	// processes that also take the lock. When running Felix inside a container, this requires the
 	// /run directory of the host to be mounted into the calico/node or calico/felix container.
 	// [Default: 0 disabled]
-	IptablesLockTimeoutSecs *int `json:"iptablesLockTimeoutSecs,omitempty"`
-	// IptablesLockProbeIntervalMillis is the time, in milliseconds, that Felix will wait between
+	IptablesLockTimeout *metav1.Duration `json:"iptablesLockTimeoutSecs,omitempty" configv1timescale:"seconds" confignamev1:"IptablesLockTimeoutSecs"`
+	// IptablesLockProbeInterval is the time, in milliseconds, that Felix will wait between
 	// attempts to acquire the iptables lock if it is not available. Lower values make Felix more
 	// responsive when the lock is contended, but use more CPU. [Default: 50]
-	IptablesLockProbeIntervalMillis *int `json:"iptablesLockProbeIntervalMillis,omitempty"`
-	// IpsetsRefreshIntervalSecs is the period, in seconds, at which Felix re-checks all iptables
+	IptablesLockProbeInterval *metav1.Duration `json:"iptablesLockProbeIntervalMillis,omitempty" configv1timescale:"milliseconds"`
+	// IpsetsRefreshInterval is the period, in seconds, at which Felix re-checks all iptables
 	// state to ensure that no other process has accidentally broken Calico’s rules. Set to 0 to
 	// disable iptables refresh. [Default: 90]
-	IpsetsRefreshIntervalSecs *int `json:"ipsetsRefreshIntervalSecs,omitempty" confignamev1:"IpsetsRefreshInterval"`
+	IpsetsRefreshInterval *metav1.Duration `json:"ipsetsRefreshIntervalSecs,omitempty" configv1timescale:"seconds" confignamev1:"IpsetsRefreshInterval"`
```

Comment:
Now that the name has reverted back to the original, we no longer need the confignamev1 tag.